client/src/components/Header_Bar/header.css

```diff
@@ -0,0 +1,67 @@
+.HeadRectangle {
```

Comment:
Please stay away from naming something *Rectangle, use things like box or container for example